lib/streamlit/type_util.py

```diff
@@ -197,13 +197,16 @@ def get_fqn_type(obj: object) -> str:
 _PANDAS_SERIES_TYPE_STR: Final = "pandas.core.series.Series"
 _PANDAS_STYLER_TYPE_STR: Final = "pandas.io.formats.style.Styler"
 _NUMPY_ARRAY_TYPE_STR: Final = "numpy.ndarray"
+_SNOWPARK_DF_TYPE_STR: Final = "snowflake.snowpark.dataframe.DataFrame"
+_SNOWPARK_DF_ROW_TYPE_STR: Final = "snowflake.snowpark.row.Row"
 
 _DATAFRAME_LIKE_TYPES: Final[tuple[str, ...]] = (
     _PANDAS_DF_TYPE_STR,
     _PANDAS_INDEX_TYPE_STR,
     _PANDAS_SERIES_TYPE_STR,
     _PANDAS_STYLER_TYPE_STR,
     _NUMPY_ARRAY_TYPE_STR,
+    _SNOWPARK_DF_TYPE_STR,
```

Comment:
Do we need this here? I'm not sure what dataframe like really means here, but is suspect that it might be about compatibility with the pandas dataframe api (thought, numpy array would be also wrongly added here) . I think this is not the case here?

Reply:
> Do we need this here? I'm not sure what dataframe like really means here, but is suspect that it might be about compatibility with the pandas dataframe api (thought, numpy array would be also wrongly added here) . I think this is not the case here?

Yeah, thanks for noticing 👍 In this PR we can rely only on `def is_snowpark_dataframe` function and it's sufficient, we might go back to this, but in the PR which will handle uncollected dataframes, but I've reverted it for now 👍